inheritance/src/main/java/inheritance/Store.java

```diff
@@ -0,0 +1,32 @@
+package inheritance;
+
+import java.util.LinkedList;
+
+public class Store implements Reviewer {
+
+    LinkedList<Reviews> restaurantsReviews = new LinkedList<>();
+    public String name;
+    public int stars;
+    public static int priceCategory;
```

Comment:
Delete the `static` here. When you make an instance variable static, it means that all instances will have the same value.